src/StyleSheet.js

```diff
@@ -30,22 +29,26 @@ export default class StyleSheet {
     this.rules = Object.create(null)
     // Used to ensure correct rules order.
     this.rulesIndex = []
-    this.classes = Object.create(null)
+    this.classes = this.options.classes = Object.create(null)
```

Comment:
I know it is  a bad thing to pass an object and then modify it in some random places ... however if an object has a very clear interface and is basically a registry, just without methods for performance reasons, not sure this is actually bad.